### DISCUSSION TITLE
True SSO (SAML w/ SCIM) integration with 3rd Party IdP?

### DISCUSSION THREAD

**Discussion by kevinjbeattie** [community/community#128427] [npm]
Hi, hoping someone at NPM can answer this question (I did not see an open issue for it), but is there any plan for NPM to support SAML based single sign-on via 3rd party IdP's like Okta or OneLogin (and SCIM)? Secure Web Authentication (SWA) is the only available option that I see in the Okta (our IdP) which is insufficient for us.

**Comment by monishcm** [comment]
We do not support SAML/SSO or other 3rd part IdP integration.
